test/simple_optimizers.jl

```diff
@@ -3,13 +3,14 @@
 N = 1000
 dim_x = 10
 dim_y = 8
-σ = 0.01
+σ = 0.0
```

Comment:
Why didn't you keep sigma > 0 ?

Reply:
When sigma > 0, the value of the `objective_gap` metric (only computed when `target == "none"`) is not entirely correct : the optimal `y` in the *learning by experience* case is `optimizer(true_model(x))`, which is noisy when `sigma > 0`. A solution to keep `sigma > 0` is to compute an `unperturbed_Y` during the `generate_dataset` and use it to compute the `objective_gap`.